addons/hr/models/res_users.py

```diff
@@ -334,3 +334,12 @@ def action_open_employees(self):
             'res_id': employees.id,
             'view_mode': 'form',
         }
+
+    def onchange(self, values, field_names, fields_spec):
```

Comment:
Overriding ORM methods for a simple use case is usually a bad idea. 

Did you try adding the field ``image_1920`` into the ``HR_WRITABLE_FIELDS`` instead ? 

Normally it should solve your use case.

Reply:
Adding `image_1920` into `HR_WRITABLE_FIELDS` and `HR_READABLE_FIELDS` does not do a trick ...

We get seemingly unrelated error when we try to change user picture:

```
Access Error

The requested operation can not be completed due to security restrictions.

Document type: Employee (hr.employee)
Operation: read
User: 6
Fields:
- private_street (allowed for groups 'Employees / Officer: Manage all employees')
```

My guess is that this is due to the changes in `onchage`. When we change something in the form view, it reads all the fields that are on the form
https://github.com/odoo/odoo/blob/40934f4089c82f76ca88173271eb10dad7309e1a/addons/web/models/models.py#L889-L898

It reads them here https://github.com/odoo/odoo/blob/40934f4089c82f76ca88173271eb10dad7309e1a/addons/web/models/models.py#L1004

I guess with that change, we should have overriden  onchange, the same way do read method? But looking back at the change we should probably have `fields_to_read = HR_READABLE_FIELDS + HR_WRITABLE_FIELDS`